doc/btp_bas.txt

```diff
@@ -0,0 +1,132 @@
+BAS Service (ID 30)
+=====================
+
+Commands and responses:
+
+	Opcode 0x00 - Error response
+
+	Opcode 0x01 - Read Supported Commands command/response
+
+		Controller Index:	<controller id>
+		Command parameters:	<none>
+		Response parameters:	<supported commands> (variable)
+
+		Each bit in response is a flag indicating if command with
+		opcode matching bit number is supported. Bit set to 1 means
+		that command is supported. Bit 0 is reserved and shall always
+		be set to 0. If specific bit is not present in response (less
+		than required bytes received) it shall be assumed that command
+		is not supported.
+
+		In case of an error, the error response will be returned.
+
+Events:
+	Opcode 0x02 - BAS Set Battery Level
+
+		Controller Index:	<controller id>
+		Event parameters:	Address_Type (1 octet)
+					Address (6 octets)
```

Comment:
Hmm this is server configuration,   do we need address here? (normally address is needed for BTP only if it is suppose to performed action in specified peer)  

I'd remove address from all those commands